### DISCUSSION TITLE
Light theme and jira TUI

### DISCUSSION THREAD

**Discussion by karampok** [ankitpokhrel/jira-cli#710] [General]
Hi, great work! I have a fish terminal with a gruvbox light theme. The table  view is okay, but when I am clicking the view ` v to view selected issue details` the output is unreadable. Is there any tip to just make viewable? Thanks

(last question 2022 answered that there is theme support, I am asking only if I can do any trick e.g. disable colors)

**Comment by ankitpokhrel** [comment]
@karampok you can use `GLAMOUR_STYLE` env to set the [theme](https://github.com/charmbracelet/glamour/tree/master/styles/gallery)

eg:
```sh
GLAMOUR_STYLE=light jira issue list
GLAMOUR_STYLE=dracula jira issue view <key>
```
